components/authn-service/cmd/migrate-tokens/migrate-tokens.go

```diff
@@ -15,7 +15,7 @@ var (
 )
 
 const (
-	tokensURL  = "https://localhost/iam/v2/tokens"         // nolint
+	tokensURL  = "https://localhost/apis/iam/v2/tokens"    // nolint
```

Comment:
woof 😅 hope no one ran into this

Reply:
I think (no data) this is very unlikely.